### DISCUSSION TITLE
Movements get bugged when...

### DISCUSSION THREAD

**Discussion by O-beep-s** [cafali/SnapKey#56] [General]
Hi, 
amazing program, but i found a bug. It happens like this: 

I hold any movement key down (wasd) and start holding "Alt" down. While holding both buttons down, i let go of the movement key. Then my character is stuck walking in the direction of the pressed key/keys. To get my character back to normal again, i need to click the according movement keys again.

I really appreciate your effort, thanks for this application!

**Comment by minteeaa** [comment]
confirmed in #52, fix in the works
